includes/Admin/Admin.php

```diff
@@ -229,4 +231,22 @@ public function prefill_post_title( $title ) {
 		// Otherwise it runs through wptexturize() and the like, which we want to avoid.
 		return $post->post_title;
 	}
+
+	
+	/**
+	 * Adds active publisher logo to media state output.
+	 *
+	 * @since 1.2.2
+	 *
+	 * @param string[] $media_states Array of media states.
+	 * @param WP_Post  $post    Post object.
+	 * @return string[] updated media states.
+	 */
+	public function media_states( $media_states, $post ): array {
+		$active_publisher_logo = absint( get_option( Settings::SETTING_NAME_ACTIVE_PUBLISHER_LOGO ) );
```

Comment:
Let's do this - https://github.com/GoogleForCreators/web-stories-wp/blob/a6fe5136095cc3d1fe8b3974c5c3db78ed0fa32d/includes/REST_API/Publisher_Logos_Controller.php#L193

Reply:
Loop over and add the label to each (not just the default)  Is that what you're thinking?

Reply:
No, it is able using the settings class here.

Reply:
Yes we can use the settings class. But not use the different setting name.